pages/home.jsx

```diff
@@ -101,13 +105,15 @@ var BlogSection = React.createClass({
     }
   },
   componentDidMount: function() {
-    var self = this;
-    blogFeedLoader(this, window, "https://blog.webmaker.org/tag/teachtheweb/feed", function(data) {
-      self.setState({
-        featuredPostData: data.featuredPostData,
-        latestPostsData: data.latestPostsData
+    this.props.loadBlogPosts(function(data) {
+      if (!this.isMounted()) {
+        return;
+      }
+      this.setState({
+        featuredPostData: data.featuredPosts,
+        latestPostsData: data.latestPosts
```

Comment:
Yep, I'm leaving that up to you b/c I could see reasons for having `Data` at the end of the props... for instance if you eventually have lots of different props and want to be very explicit that these ones just contain data... although I guess all props should only contain data... umm anyways I can change it in this PR, or you could merge it and change it yourself, whatevs works!